pkg/backtest/exchange.go

```diff
@@ -136,11 +136,11 @@ func (e *Exchange) resetMatchingBooks() {
 	e.matchingBooksMutex.Unlock()
 }
 
-func (e *Exchange) addMatchingBook(symbol string, market types.Market) {
-	e.matchingBooksMutex.Lock()
-	e._addMatchingBook(symbol, market)
-	e.matchingBooksMutex.Unlock()
-}
+// func (e *Exchange) addMatchingBook(symbol string, market types.Market) {
+// 	e.matchingBooksMutex.Lock()
+// 	e._addMatchingBook(symbol, market)
+// 	e.matchingBooksMutex.Unlock()
+// }
```

Comment:
Can simply remove it? but the originally I added this method is for the race condition (it's still in the middle of the refactoring)

Reply:
removed